sound/soc/intel/boards/sof_sdw.c

```diff
@@ -1483,6 +1487,9 @@ static int sof_card_dai_links_create(struct device *dev,
 	for (i = 0; i < SDW_MAX_GROUPS; i++)
 		group_generated[i] = false;
 
+	for (i = 0; i < SDW_MAX_LINKS; i++)
+		sdw_pin_index[i] = SDW_INTEL_BIDIR_PDI_BASE;
```

Comment:
This is so hard to follow. This isnt really pin index is all links have the same value. Its really link_pin_index_base, right?

Reply:
to be clearer @ranj063: Pin0/1 are reserved for "bulk", a SoundWire mechanism where command data can be transmitted over data ports. All audio transfers are handled starting with Pin2+.